Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -200,5 +195,34 @@ private BocList GetControlToValidate ()
 
       return bocListControl;
     }
+
+    void IValidatorWithDynamicErrorMessage.RefreshErrorMessage ()
+    {
+      var bocList = GetControlToValidate();
+      var validationRepository = bocList.ValidationFailureRepository;
+
+      var sb = new StringBuilder();
+
+      var validationFailuresForBusinessObjectPropertyOfBocList = validationRepository.GetUnhandledValidationFailuresForBocList(true);
+      foreach (var validationFailure in validationFailuresForBusinessObjectPropertyOfBocList)
+      {
+        sb.AppendLine(validationFailure.Failure.ErrorMessage);
+      }
+
+      var hasRowsWithUnhandledValidationFailures = validationRepository.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(true).Any();
+      if (hasRowsWithUnhandledValidationFailures)
+      {
+        var remainingValidationFailuresText = GetControlToValidate()
+            .GetResourceManager()
+            .GetString(BocList.ResourceIdentifier.RemainingValidationFailureText);
+
+        sb.AppendFormat(remainingValidationFailuresText);
```

Comment:
```suggestion
        sb.AppendLine(remainingValidationFailuresText);
```
For consistency, we should use AppendLine here, too. Otherwise, some isntance have a trailing new line and others don't. We do not need AppendFormat since there's no parameters.